### PR TITLE
#43 ft(blog): add create blog firebase connections

### PR DESCRIPTION
**Pull request by SimonImanigirimpuhwe**
#### What does this PR do?

- Store created blog to Firebase

#### Description of Task to be completed?

- Store blog post data to Cloud firestore

- store post image to the Firestore

#### How should this be manually tested?

- Clone the repo

- In the directory checkout this branch `ft-create-article-#43`

- Go to localhost by firstly switch the live server in your code editor

- If you have account go to the login page and provide your credentials and get logged in

- Once logged in get redirected to the dashboard and click on `New article` button and you should see the form and fill the inputs and send

> You should see a successful message when blog is created or failure if blog is not saved

#### Any background context you want to provide?

-To be able to create a blog you must first login

#### What are the relevant Trello stories?

- [#43](https://trello.com/c/k1JxRMBR/43-create-article)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

**Comment by SamuelSabasNayo** [comment]
You can replace innerHTML by creating elements as Mike told us.

File: scripts/admin.js [hunk 1]
```diff
@@ -248,8 +249,11 @@ const displayEmail = domElement("#admin-display-email");
 
 auth.onAuthStateChanged(user => {
   if (user){
+    console.log()
```
Comment (Anguandia) [inline scripts/admin.js:252]: Do not log to console except for testing in development****

File: scripts/login.js [hunk 2]
```diff
@@ -69,8 +69,8 @@ async function loginFnc(email, password, form) {
 
 //check success
 function checkSuccess(form) {
-  location.href = '../admin/';
   if (isLogged) {
+    location.href = '../admin/';
```
Comment (Anguandia) [inline scripts/login.js:73]: good auto redirect

**Review by jemmyarina — COMMENTED** [review]
LOOKS GOOD. you can merge

**Pull request merged**